keyboards/pistachio_mp/keymaps/via/keymap.c

```diff
@@ -63,12 +66,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
 #ifdef ENCODER_ENABLE
 bool encoder_update_user(uint8_t index, bool clockwise) {
+    int l = get_highest_layer(layer_state);
     if (index == 0) {
-        // Volume control
         if (clockwise) {
-            tap_code(KC_VOLU);
+            tap_code16(keymap_key_to_keycode(l, ENC_CW));
```

Comment:
This will miss most keycodes. 

You should use action_exec here instead, as that will work with all keycodes. 

See `keyboards/work_louder/encoder_actions.c` for an example of what I mean.